CONTRIBUTING.md

```diff
@@ -51,55 +122,116 @@ path_base_for_github_subdir:
 --->
 ```
 
-When creating new pages, you should provide the `linkTitle` attribute. This is used
-to generate the navigation bar on the website, and will be listed relative to the
-"parent" document.
+When creating new markdown files, you should provide the `linkTitle` attribute.
+This is used to generate the navigation bar on the website,
+and will be listed relative to the "parent" document.
 
-## Automation
+### 3. Verify the changes before committing
 
-Semantic Conventions provides a set of automated tools for general development.
+Before sending a PR with your changes, make sure to run the automated checks:
 
-### Consistency Checks
+```bash
+make check
+```
 
-The Specification has a number of tools it uses to check things like style,
-spelling and link validity. Before using the tools:
+Alternatively, you can run each check individually.
+Refer to the [Automation](#automation) section for more details.
 
-- Install the latest LTS release of **[Node](https://nodejs.org/)**.
-  For example, using **[nvm][]** under Linux run:
+### 4. Changelog
 
-  ```bash
-  nvm install --lts
-  ```
+#### When to add a Changelog Entry
 
-- Install tooling packages:
+Pull requests that contain user-facing changes will require a changelog entry.
+Keep in mind the following types of users (not limited to):
 
-  ```bash
-  npm install
-  ```
+1. Those who are consuming the conventions (e.g. in alerts, dashboards, queries)
+2. Those who are using the conventions in instrumentations (e.g., library authors)
+3. Those who are using the conventions to derive heuristics, predictions and automatic analyses (e.g. observability products/back-ends)
+
+If a changelog entry is not required (e.g. editorial or trivial changes),
+a maintainer or approver will add the `Skip Changelog` label to the pull request.
+
+##### Examples
+
+Changelog entry required:
+
+- Any modification to existing conventions with change in functionality/behavior
+- New semantic conventions
+- Changes on definitions, normative language (in `/docs`)
+
+Judgement call:
+
+No changelog entry:
+
+- Typical documentation/editorial updates (e.g. grammar fixes, restructuring)
+- Changes in internal tooling (e.g. make file, GH actions, etc)
+- Refactorings with no meaningful change in functionality
+- Chores, such as enabling linters, updating dependencies
+
+#### Adding a Changelog Entry
+
+The [CHANGELOG.md](./CHANGELOG.md) files in this repo is autogenerated
+from `.yaml` files in the `./.chloggen` directory.
+
+Your pull-request should add a new `.yaml` file to this directory.
```

Comment:
maybe add an example or guide for naming? Like PR or issue number?

Reply:
the task `make chlog-new` will create the file based on the branch name, so you shouldn't worry about naming the file :)

Reply:
See : 6ecc530